packages/application-shell/src/components/app-bar/app-bar.tsx

```diff
@@ -115,7 +116,11 @@ const AppBar = (props: Props) => {
           display: flex;
         `}
       >
-        <div id={CONTAINERS.LEFT_OF_PROFILE}></div>
+        <div id={CONTAINERS.LEFT_OF_PROFILE}>
+          <WorkspacesNavigationButton
+            projectKey={props.projectKeyFromUrl || previousProjectKey}
+          />
+        </div>
```

Comment:
This element is used as a portal container (we currently use it to render the staff bar). Probably we shouldn't define an element inside of it.

Reply:
Thanks for that context! a41a4588e8dd0e8689e8a068acfb140ad9ddaae3